source/content/drupal-9.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Drupal 9 Early Access
+description: Test the latest Drupal on the Pantheon Website Management Platform.
+categories: [get-started]
+tags: [site]
+earlyaccess: true
+contributors: [populist, edwardangert]
+reviewed: "2020-05-21"
+---
+
+Drupal 9 is, according to [drupal.org](https://www.drupal.org/docs/understanding-drupal/drupal-9-resources), "a cleaned-up version of Drupal 8... \[with] deprecated code removed and third-party dependencies updated."
+
+Drupal 9 updates Drupal’s underlying dependencies like [Symfony 4.4](https://symfony.com/releases/4.4) and [Twig 2](https://twig.symfony.com/doc/2.x/index.html), removes several deprecated API functions in favor of better options, and allows everyone running Drupal 8.8+ an easy upgrade path to Drupal 9 & beyond.
+
+<Alert title="Warning" type="danger">
+
+**Do not use this, or any, Early Access release for your live site.**
+
+Currently, Drupal 9 is only available on Pantheon as an Early Access feature release. This means the feature is currently in active development and is intended for testing and evaluation purposes only.
+
+Pantheon engineers are rolling out changes often, and easy updates from Drupal 9 Early Access are not guaranteed.
+
+**Sites may be deleted without notice**.
+
+</Alert>
+
+Choose one of the following methods to get started with a new Drupal 9 site to [create a new Drupal 9 site with integrated Composer](#create-a-new-drupal-9-site-with-integrated-composer) or to test your existing site's [compatibility with an upgrade](#test-an-existing-drupal-site-for-drupal-9-upgrade-compatibility).
+
+## Create a New Drupal 9 Site with Integrated Composer
+
+Join our Early Access program to test out both Drupal 9 and Pantheon-native Composer support.
+
+1. [Fill out this form](https://docs.google.com/forms/d/1lahWKMT2VHXfr9hg15VIQY2Kn6z_j77o7Te6hZqsNgw) to get access to our “Drupal 9 Early Access” group. Once you are added, you will access to a new "Drupal 9 Early Access" site creation option.
+
+1. Spin up a new "Drupal 9 Early Access" site with `ic-demo-2020-` as a site name prefix (i.e., `ic-demo-2020-beta3-d9-test`). This will give you [Drupal 9 starter codebase](https://github.com/stevector/drupal-9-project) and will automatically trigger a Composer install and `git commit` on site creation.
```

Comment:
@stevector do you still want this and L91 pointing to github/stevevector?

Reply:
Yeah, I'm fine with leaving it since this is the Early Access repo and won't be used  long-term. We can move the repo later if we need to and GitHub will redirect.